src/URLNodeList.jsx

```diff
@@ -62,7 +62,7 @@ export default function URLNodeList(props) {
             toast.warning(`Processing this link did not work: ${errorUrl}`, {
 				position: toast.POSITION.BOTTOM_CENTER,
 			});
```

Comment:
React didn't yell at us for using a hardcoded key? These keys are suppose to help React keep track of elements in a list so if any UrlNode is rendered twice with the same key, React tends to be upset. Usually its best to keep the key the same as an index in the list or something like that, but if it works we can probably keep it.